mempool/mempool_test.go

```diff
@@ -66,7 +65,10 @@ func checkTxs(t *testing.T, mempool *Mempool, count int) types.Txs {
 			t.Error(err)
 		}
 		if err := mempool.CheckTx(txBytes, nil); err != nil {
-			t.Fatalf("Error after CheckTx: %v", err)
+			if IsPreCheckError(err) {
```

Comment:
why are these happening here?

Reply:
TestMempoolFilters will fail otherwise (it asserts a number of txs created, not panic)